data-management/event-logging.js

```diff
@@ -0,0 +1,49 @@
+const {RequestEvent} = require("../bento-event-logging/model/request-event");
+const {RegistrationEvent} = require("../bento-event-logging/model/registration-event");
+const {ReviewEvent} = require("../bento-event-logging/model/review-event");
+const {UpdateEvent} = require("../bento-event-logging/model/update-event");
+const {getArmsFromArmIds, logEventNeo4j, getUserByEmailIDP} = require("./neo4j-service");
+const {PENDING, APPROVED, REVOKED} = require("../bento-event-logging/const/access-constant");
+
+
+const logRequestArmAccess = async (armIDs, userID, userEmail, userIDP) => {
+    const arms = await getArmsFromArmIds(armIDs);
+    for (const arm of arms) {
+        const requestEvent = new RequestEvent(arm.properties.id, arm.properties.name, userID, userEmail, userIDP);
+        await logEventNeo4j(requestEvent);
```

Comment:
I was wondering if you can implement bulk insert functionality. Is it possible to insert multiple nodes at once, rather than individually?

I researched regarding bulk insert below;
CALL apoc.cypher.runMany(
  'CREATE (n:Event {event_id:$eventid});
   CREATE (n:Event {event_id:$eventid2});
',
  {eventid:"1", eventid2:"2"}
);